run-tests.sh

```diff
@@ -10,5 +10,5 @@ pydocstyle reana_db && \
 isort -rc -c -df **/*.py && \
 check-manifest --ignore ".travis-*" && \
 sphinx-build -qnNW docs docs/_build/html && \
-python setup.py test && \
+REANA_SQLALCHEMY_DATABASE_URI=sqlite:///test.db python setup.py test && \
```

Comment:
```suggestion
REANA_SQLALCHEMY_DATABASE_URI=sqlite:// python setup.py test && \
```
So it doesn't create the file.